src/sentry/sentry_apps/tasks/sentry_apps.py

```diff
@@ -270,6 +270,28 @@ def _process_resource_change(
         ]
 
         for installation in installations:
+            try:
+                service_hook = ServiceHook.objects.get(
+                    organization_id=org.id,
+                    actor_id=installation.id,
```

Comment:
not sure why `actor_id=` and not `installation_id=` but that's how we do it in `send_webhooks` (line 520)